lib/alephant/broker/response/batch.rb

```diff
@@ -12,7 +12,7 @@ class Batch < Base
         def initialize(components, batch_id, request_env)
           @components = components
           @batch_id   = batch_id
-          @status     = component_not_modified(batch_response_headers, request_env) ? 304 : 200
+          @status     = Alephant::Broker::Response::Base.component_not_modified(batch_response_headers, request_env) ? NOT_MODIFIED_STATUS_CODE : 200
```

Comment:
Ah that's it. I was using `self.`, and it said there was no method `component_not_modified` on it. That does seem better, as Base is quite specific for the class to include.